_VERSION_MIGRATION/migrate-to-next.py

```diff
@@ -98,11 +99,13 @@ def get_containers(containers):
 
 def get_spec(config):
     containers = [config]
+    terminationGracePeriod = "100s"
     if config.get('containers'):
         containers = config['containers']
-
+    if "terminationGracePeriod" in config:
+        terminationGracePeriod = config['terminationGracePeriod']
     return {
-        'terminationGracePeriod': 100,
+        'terminationGracePeriod': terminationGracePeriod,
```

Comment:
Hey @manuellysuzik you need to append the character `s` since in the maestro v9 the `shutdownTimeout` is an integer.